App_Data/SwaggerSpecs/Microsoft.GuestConfiguration/guestconfiguration.json

```diff
@@ -1,1285 +1,1281 @@
-              "in": "path",
-              "required": true,
-              "type": "string",
-              "description": "Name of the guest configuration assignment."
-            },
-            {
-              "name": "parameters",
-              "in": "body",
-              "required": true,
-              "schema": {
-                "$ref": "#/definitions/GuestConfigurationAssignment"
-              },
-              "description": "Parameters supplied to the create or update guest configuration assignment."
-            },
-            {
-              "$ref": "../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
-            },
-            {
-              "$ref": "../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
-            },
-            {
-              "$ref": "./definitions.json#/parameters/VmNameParameter"
-            },
-            {
-              "$ref": "../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
-            }
-          ],
-          "responses": {
-            "201": {
-              "description": "Created. The guest configuration assignment for the VM was created successfully.",
-              "schema": {
-                "$ref": "#/definitions/GuestConfigurationAssignment"
-              }
-            },
-            "200": {
-              "description": "OK. The guest configuration assignment for the VM was updated successfully.",
-              "schema": {
-                "$ref": "#/definitions/GuestConfigurationAssignment"
-              }
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GuestConfigurationAssignment"
             },
-            "default": {
-              "description": "Error response describing why the operation failed.",
-              "schema": {
-                "$ref": "../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
-              }
-            }
+            "description": "Parameters supplied to the create or update guest configuration assignment."
           },
-          "x-ms-long-running-operation": true,
-          "produces": [
-            "application/json"
-          ],
-          "consumes": [
-            "application/json"
-          ]
-        },
-        "get": {
-          "tags": [
-            "GuestConfigurationAssignments"
-          ],
-          "operationId": "GuestConfigurationAssignments_Get",
-          "description": "Get information about a guest configuration assignment",
-          "x-ms-examples": {
-            "Get a guest configuration assignment": {
-              "$ref": "./examples/getGuestConfigurationAssignment.json"
-            }
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
```

Comment:
common types are available at a different relative path (you can see where the old file was referencing parameters/SubscriptionIdParameter from as an example)